ls1intum/Artemis
# Bugfixes: re-evaluate mc-quiz 2

### Checklist

### Motivation and Context


Follow-up PR to https://github.com/ls1intum/Artemis/pull/1515
After saving changes and changing thing again 'pending changes' isn't shown and therefore the later changes can't be saved.

### Description

Use lodash to clone objects in order to prevent references being assigned.
(And also delete an empty file)

### Steps for Testing


1. Log in to Artemis
2. Create a mc-quiz which is ready to re-evaluate.
3a. Change the text (or an answer option)
3b. 'pending changes' is shown
3c. save your changes
3d. 'pending changes' isn't shwon
4a. Change the text (or an answer option)
4b. 'pending changes' is shown
4c. save your changes
4d. 'pending changes' isn't shwon
(optional: repeat 4a-4d until bored or convinced you can save changes as often as you want)
